templates/zerver/help/desktop-app-install-guide.md

```diff
@@ -0,0 +1,73 @@
+# How to install Desktop app
+
+[LR]: https://github.com/zulip/zulip-electron/releases/latest
+
+If you download from the [releases page][LR], be careful what version you pick.
+There are two versions available -
+
+- **beta:** these releases are the right balance between getting
+    new features early while staying away from nasty bugs.
+- **stable:** these releases are more thoroughly tested; they receive
+    new features later, but there's a lower chance that things will go wrong.
+
+## MAC
```

Comment:
If I hyperlink this using -
`[MAC](help/desktop-app-install-guide#mac)`

it throws - 
```sh
(zulip-py3-venv)vagrant@vagrant-ubuntu-trusty-64:/srv/zulip$ ./tools/test-help-documentation.py
Waiting for test server...........................................................................................................................................................................................................................................................................................
2017-09-19 22:24:12 [scrapy.core.scraper] ERROR: Spider error processing <GET http://localhost:9981/help/desktop-app-install-guide#mac> (referer: http://localhost:9981/help/desktop-app-install-guide)
Traceback (most recent call last):
  File "/srv/zulip-venv-cache/eb52c7dd5a2cf88a54ada01947af4a1a6679a3e4/zulip-py3-venv/lib/python3.4/site-packages/twisted/internet/defer.py", line 653, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/srv/zulip/tools/documentation_crawler/documentation_crawler/spiders/common/spiders.py", line 68, in check_permalink
    "Permalink #{} is not found on page {}".format(permalink, response.request.url))
Exception: Permalink #mac is not found on page http://localhost:9981/help/desktop-app-install-guide#mac

Failed
```
Is there any workaround for this?